Code/Tools/AssetProcessor/Platform/Linux/native/FileWatcher/FileWatcher_linux.cpp

```diff
@@ -98,7 +98,7 @@ void FileWatcher::PlatformImplementation::AddWatchFolder(QString folder, bool re
             [[maybe_unused]] const auto err = errno;
             [[maybe_unused]] AZStd::fixed_string<255> errorString;
             AZ_Warning("FileWatcher", false, "inotify_add_watch failed for path %s: %s", dirName.toUtf8().constData(), strerror_r(err, errorString.data(), errorString.capacity()));
-            return;
+            continue;
```

Comment:
Seems like we should continue only if errno matches [EACCES](https://man7.org/linux/man-pages/man2/inotify_add_watch.2.html) which indicates a permission problem